src/averageAges.js

```diff
@@ -1,61 +1,60 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = people) {
+  let calculateCentury = century;
+
+  if (Number(century)) {
+    calculateCentury = people
+      .filter(peopleCentury => Math.ceil(peopleCentury.died / 100) === century);
+  }
```

Comment:
looks confusing, in the century we can have either `number` or `array`, let's change this solution so it'd be not that confusing